### PR TITLE
Aggregate analyzers metadata

### PR DESCRIPTION
**Pull request by ybiquitous**
This aggregates all analyzers' metadata to a YAML file.
By this metadata file, we will be able to make some automation (e.g. an auto-generated README section).

### CODE REVIEW

**Comment by ybiquitous** [comment]
@yykamei I want to listen to your opinion. 🙏

File: lib/tasks/bump/analyzers.rb [hunk 1]
```diff
@@ -46,20 +47,9 @@ def initialize(response)
...
-    javasee:  { name: "JavaSee", github: "sider/JavaSee" },
-    misspell: { name: "Misspell", github: "client9/misspell" },
-    shellcheck: { name: "ShellCheck", github: "koalaman/shellcheck" },
-    swiftlint: { name: "SwiftLint", github: "realm/SwiftLint" },
-  }.freeze
-
   def self.each
-    ANALYZERS_UNSUPPORTED_BY_DEPENDABOT.each do |analyzer, meta|
+    analyzers = YAML.safe_load(Pathname(__dir__).join("../../../analyzers.yml").read, symbolize_names: true).fetch(:analyzers)
+    analyzers.each do |analyzer, meta|
```
Comment (yykamei) [inline lib/tasks/bump/analyzers.rb]: Is it OK to walk all analyzers here? Before this patch, only `ANALYZERS_UNSUPPORTED_BY_DEPENDABOT` seemed iterated.

Comment (ybiquitous) [reply lib/tasks/bump/analyzers.rb]: Oh, my bad. 😖 
I'll fix it soon, thank you!

Comment (ybiquitous) [reply lib/tasks/bump/analyzers.rb]: Fixed via 98c555f.

**Comment by ybiquitous** [comment]
I think this metadata file can declaratively provide data such as the following `analyzer_name`:

https://github.com/sider/runners/blob/12c1b856673ca560e7ed1ab0c837c925a18977e8/lib/runners/processor.rb#L51-L53

**Pull request merged**